sonar-ballerina-plugin/src/main/resources/io/ballerina/sonar/Ballerina1.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Avoid checkpanic, prefer explicit error handling using check keyword instead!",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "2min"
```

Comment:
Is this an estimate to fix the issue?

Reply:
I looked it up and this is the time to fix the issue. We have to provide it in minutes.

Reply:
2 minutes feels fairly low, right? Because often check-ed errors have to be handled wherever applicable once this change is done.

Reply:
I agree. How about 5 mins since this is a simple error?

Reply:
Increased to 5 mins with d42c5d59884f959feffce4a22dff4d00a3985afe